module_4/task_1/free_disk_space.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+threshold=1000
+
+checkFreeSpace() {
+  while true
+  do
+      local freeSpace=$(free --mega | grep 'Mem' | awk '{print $4}')
```

Comment:
The script should watch free space of the hard disk, so It will be better to use more descriptive variable names, like **freeDiskSpace**.